File: server/app/schemas/game.py
from pydantic import BaseModel


class Game(BaseModel):
    id: int
    name: str
    year: int
    thumbnail: str
    primary_color: str
    is_released: bool

    class Config:
        orm_mode = True


class CreateGame(BaseModel):
    name: str
    year: int
    thumbnail: str
    primary_color: str
    is_released: bool

File: pact/javascript/schemas/game.js
const { Matchers } = require("@pact-foundation/pact");
const { string, integer, boolean, term, hexadecimal } = Matchers

const gameSchema = {
  id: integer(1),
  name: string('Game 1'),
  year: integer(2021),
  thumbnail: term({
    generate: 'http://localhost/static/image.png',
    matcher: 'http://localhost/static/.*\.(png|jpg|jpeg)',
  }),
  primary_color: hexadecimal('ff0000'),
  is_released: boolean(true),
}

module.exports = gameSchema
